docs/topics/proc_add-debug-logging-to-app-vertx.adoc

```diff
@@ -0,0 +1,59 @@
+
+[#_add_debug_logging_to_application_vertx]
+= Adding log output to your {VertX} application.
+
+. To add logging for your application, define a method to access the `Logger` class:
+To do that you use `LoggerFactory` class to get an instance of `Logger` which you then use for logging, for example:
++
+[source,java,options="nowrap",subs="attributes+"]
+--
+Logger logger = LoggerFactory.getLogger(className);
+
+logger.info("something happened");
+logger.error("oops!", exception);
+logger.debug("debug message");
+logger.warn("warning");
+--
++
+[CAUTION]
+--
+Logging backends use different formats to represent replaceable tokens in parameterized messages.
+As a consequence, if you rely on Vert.x parameterized logging methods, you will not be able to switch logging backends without changing your code.
+--
+
+////
```

Comment:
This is a comment for reference purposes, will be removed before finalization, please ignore the wall of text.